stampit.js

```diff
@@ -189,16 +191,21 @@ isStamp = function isStamp(obj) {
  * @return {Function}             A composable stampit factory
  *                                (aka stamp).
  */
-convertConstructor = function convertConstructor(Constructor) {
-  return stampit().methods(Constructor.prototype).enclose(Constructor);
-};
+function convertConstructor(Constructor) {
+  var stamp = stampit();
+  mixer.mixInChainFunctions(stamp.fixed.methods, Constructor.prototype);
+  mixer.mixIn(stamp.fixed.state, Constructor);
```

Comment:
If it converts classic constructors partially then we have to rename the function to `convertSomePropertiesOfTheConstructorButNotAll`. :)))

People usually have no idea if a constructor has some static props. For example `mongoose.Model`. It's got them all.

Sample code:
```js
function Constructor() {}
Constructor.clone = function (constructorObject) { ...
```

What's your proposal?